src/snowcli/cli/common/decorators.py

```diff
@@ -132,6 +133,12 @@ def wrapper(**options):
         annotation=Optional[bool],
         default=DebugOption,
     ),
+    inspect.Parameter(
+        "temporary_connection",
```

Comment:
Shouldn't it be a part of `GLOBAL_CONNECTION_OPTIONS` instead of `GLOBAL_OPTIONS`?

Reply:
Moved